docs/layer_reference/utility.rst

```diff
@@ -32,13 +32,6 @@ Print Layer
     :members:
     :undoc-members:
 
-Scale Gradient Layer
---------------------
-
-.. autoclass:: TFNetworkLayer.ScaleGradientLayer
-    :members:
-    :undoc-members:
-
```

Comment:
Why was this removed?

Reply:
I removed this because it caused an error, I did not notice this was a typo only.

Reply:
Ok, can you maybe put it in again then?